### PR TITLE
[1863] Added basic double click functionality to plugin list

### PR DESCRIPTION
**Pull request by stanica**
Double clicking a plugin will place the respective event to the left of the scrubber.

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/78eeab6d3d61790/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/78eeab6d3d61790/output.txt

Total script time: 1.00 mins

Server is running on http://pr776.butyr.org/templates/basic/
Run the Test Suite: http://pr776.butyr.org/test/

File: src/plugin/plugin-list.js [hunk 1]
```diff
@@ -44,6 +44,15 @@ define( [ "util/dragndrop", "util/lang", "editor/editor", "text!layouts/plugin-l
         }
       });
 
+      element.addEventListener("dblclick", function( e ) {
```
Comment (k88hudson) [inline src/plugin/plugin-list.js]: spacing, add a space before "dblclick"

File: src/plugin/plugin-list.js [hunk 2]
```diff
@@ -44,6 +44,15 @@ define( [ "util/dragndrop", "util/lang", "editor/editor", "text!layouts/plugin-l
...
+        var media = butter.currentMedia;
+         media.tracks[0].view.dispatch( "plugindropped", {
```
Comment (k88hudson) [inline src/plugin/plugin-list.js]: `[ 0 ]`

Comment (k88hudson) [inline src/plugin/plugin-list.js]: Also, is this going to work properly with our new trackevent overlapping stuff? I would think not

**Pull request closed without merging**